specification/common/index.bs

```diff
@@ -3161,6 +3003,8 @@ notes in a swung triplet rhythm instead:
 ```
 </div>
 
+TODO: `<beamed>` no longer exists, so the previous example is invalid.
```

Comment:
Could we just remove the previous example and then reword the introductory text to the next example? I think that should still cover the use of the performance-event while cleanly removing the beamed element from the spec.

Reply:
@mdgood Thanks — I've just updated the pull request to handle that. Look OK?